ionc/ion_timestamp.c

```diff
@@ -1307,6 +1309,7 @@ iERR ion_timestamp_binary_read(ION_STREAM *stream, int32_t len, decContext *cont
 timestamp_is_finished:
     // now we set the "has timezone" bit
     if (has_offset) {
+        ptime->tz_offset = offset;
```

Comment:
Nope, that's superfluous. Not sure how it got there -- maybe leftover from a merge or something. Will remove.